spec/requests/file_upload_spec.rb

```diff
@@ -31,19 +34,20 @@
       end
 
       before do
+        s3.stub_responses(
+          :head_object,
+          'NotFound',
+          Aws::S3::Types::HeadObjectOutput.new(last_modified: now)
+        )
```

Comment:
This is how you stub two calls to the same method sequentially. The first call will return `NotFound` and the second call returns the `HeadObjectOutput` object.